lint.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -e
+
+if [ -f venv/bin/activate ]; then
+  source venv/bin/activate
+fi
+
+check_arg=""
+if [[ "$1" = "--ci" ]]; then
+    check_arg="--check"
+fi
+
+isort --settings-file .isort.cfg .
```

Comment:
Which version of `isort` are you using? New version uses `--settings-path` and requires `-rc`

Reply:
VERSION 5.8.0.

Will update both

Reply:
```
/home/jong/.local/lib/python3.8/site-packages/isort/main.py:1206: UserWarning: W0501: The following deprecated CLI flags were used and ignored: -rc!
  warn(
/home/jong/.local/lib/python3.8/site-packages/isort/main.py:1210: UserWarning: W0500: Please see the 5.0.0 Upgrade guide: https://pycqa.github.io/isort/docs/upgrade_guides/5.0.0/
  warn(
```

not adding `-rc`.

Reply:
`--recursive` otherwise it doesn't recurse

Reply:
It's not a flag to 5.8.0
```
$ isort --help | grep -2 -- recursive
                        against.
  --dont-follow-links   Tells isort not to follow symlinks that are
                        encountered when running recursively.
  --filename FILENAME   Provide the filename associated with a stream.
```